include/z64collision_check.h

```diff
@@ -349,28 +349,28 @@ typedef struct {
 #define OC2_TYPE_2 OC1_TYPE_2 // Has OC type 2
 #define OC2_FIRST_ONLY (1 << 6) // Skips AC checks on elements after the first collision. Only used by Ganon
 
-#define TOUCH_NONE 0 // No flags set. Cannot have AT collisions
-#define TOUCH_ON (1 << 0) // Can have AT collisions
-#define TOUCH_HIT (1 << 1) // Had an AT collision
-#define TOUCH_NEAREST (1 << 2) // If a Quad, only collides with the closest bumper
-#define TOUCH_SFX_MASK (3 << 3)
-#define TOUCH_SFX_NORMAL (0 << 3) // Hit sound effect based on AC collider's type
-#define TOUCH_SFX_HARD (1 << 3) // Always uses hard deflection sound
-#define TOUCH_SFX_WOOD (2 << 3) // Always uses wood deflection sound
-#define TOUCH_SFX_NONE (3 << 3) // No hit sound effect
-#define TOUCH_AT_HITMARK (1 << 5) // Draw hitmarks for every AT collision
-#define TOUCH_DREW_HITMARK (1 << 6) // Already drew hitmark for this frame
-#define TOUCH_UNK7 (1 << 7) // Unknown purpose. Used by some enemy quads
-
-#define BUMP_NONE 0 // No flags set. Cannot have AC collisions
-#define BUMP_ON (1 << 0) // Can have AC collisions
-#define BUMP_HIT (1 << 1) // Had an AC collision
-#define BUMP_HOOKABLE (1 << 2) // Can be hooked if actor has hookability flags set.
-#define BUMP_NO_AT_INFO (1 << 3) // Does not give its info to the AT collider that hit it.
-#define BUMP_NO_DAMAGE (1 << 4) // Does not take damage.
-#define BUMP_NO_SWORD_SFX (1 << 5) // Does not have a sound effect when hit by player-attached AT colliders.
-#define BUMP_NO_HITMARK (1 << 6) // Skips hit effects.
-#define BUMP_DRAW_HITMARK (1 << 7) // Draw hitmark for AC collision this frame.
+#define ATELEM_NONE 0 // No flags set. Cannot have AT collisions
+#define ATELEM_ON (1 << 0) // Can have AT collisions
+#define ATELEM_HIT (1 << 1) // Had an AT collision
```

Comment:
I personally dont like smashing words together for macro/define names, and would be okay with `AT_ELEM_X` for example, but this kind of thing is already preveleant all over the codebase already. so i guess its a matter of preference

Reply:
There already are `AT_` flags (and ac/oc), I think `ATELEM_` is more readable as less easy to confuse with an `AT_` flag than `AT_ELEM_` would be

Besides this point (which weighs more to me than other considerations, since collider data is complicated) I agree at and elem should be separate